web/source/resources/osk/kmwosk.css

```diff
@@ -388,18 +388,18 @@ div.android #keytip {background-color:#f00;}
 #kmw-language-menu .current {font-weight:bold;}
 
 /* Popup key styles */
-#kmw-popup-keys {position:absolute;display:block;width:auto;height:auto;overflow-y:visible;
+#kmw-popup-keys {position:relative;display:block;width:auto;height:auto;overflow-y:visible;
```

Comment:
Ah, that's a good call.  More relative, within-OSK positioning is definitely a plus.

... and now that I think of it, that probably addresses my concern in the other comment; we won't need to track any offset-ancestry further back if we always offset from `_Box`.